index.js

```diff
@@ -491,8 +491,8 @@ class NodeClam {
 
         try {
             await fs_access(path, fs.constants.R_OK);
-
-            const {stdout} = await cp_exec(version_cmds[scanner]);
+            version_cmds_exec = version_cmds[scanner].split(' ');
+            const {stdout} = await cp_execfile(version_cmds_exec[0], [version_cmds_exec[2]]);
```

Comment:
```suggestion
            const {stdout} = await cp_execfile(version_cmds_exec[0], [version_cmds_exec[1]]);
```

Reply:
I'm okay with the change but the code isn't going to work since there will never been a 2nd index for this command since the whole command is just `/some/path/to/clamav/binary --version`. If you split that by `'  '`, you'll get `['/some/path/to/clamav/binary', '--version']`.

Please correct and I'll merge the PR.